cnxarchive/sql/search/title.part.sql

```diff
@@ -10,4 +10,4 @@ SELECT
 FROM
   latest_modules
 WHERE
-  name ~* req(%({0})s::text)
+  StripHTML(name) ~* req(%({0})s::text)
```

Comment:
Actually creating an index like this is not going to help if we use `LIKE` in our query.... I'm going to have a look at what kind of index we need.